test/Confluent.Kafka.IntegrationTests/Tests/AssignOverloads.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Text;
+using System.Collections.Generic;
+using Confluent.Kafka.Serialization;
+using Xunit;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public static partial class Tests
+    {
+        /// <summary>
+        ///     Simple test of both Consumer.Assign overloads.
+        /// </summary>
+        [Theory, MemberData(nameof(KafkaParameters))]
+        public static void AssignOverloads(string bootstrapServers, string topic)
+        {
+            var consumerConfig = new Dictionary<string, object>
+            {
+                { "group.id", "u-bute-group" },
```

Comment:
So disappointed it wasn't a reference to http://www.ultimatebbqs.com.au/products-page/accessories/u-bute-portable-butane-power-bbq/